pkg/rancher/rancher.go

```diff
@@ -451,7 +451,7 @@ func (r *Rancher) Start(ctx context.Context) error {
 		return nil
 	})
 
-	r.Wrangler.OnLeader(func(ctx context.Context) error {
+	r.Wrangler.OnLeaderOrDie("rancher-start", func(ctx context.Context) error {
```

Comment:
I think after full fetch of `main` and rebase this would be the only change expected.